src/components/Picker/BasePicker.js

```diff
@@ -12,305 +12,326 @@ import themeColors from '../../styles/themes/default';
-                        {this.props.value}
-                    </Text>
-                    {Boolean(this.props.hintText)
-                    && (
-                        <Text style={[styles.textLabel, styles.colorMuted, styles.mt2]}>
-                            {this.props.hintText}
-                        </Text>
-                    )}
-                </View>
-            );
-        }
-
-        return (
-            <>
-                <View
-                    ref={el => this.root = el}
-                    style={[
-                        styles.pickerContainer,
-                        this.props.isDisabled && styles.inputDisabled,
-                        ...this.props.containerStyles,
-                        this.state.isHighlighted && styles.borderColorFocus,
-                        hasError && styles.borderColorDanger,
-                    ]}
-                >
-                    {this.props.label && (
-                        <Text pointerEvents="none" style={[styles.pickerLabel, styles.textLabelSupporting]}>
-                            {this.props.label}
-                        </Text>
-                    )}
-                    <RNPickerSelect
-                        onValueChange={this.onInputChange}
-
-                        // We add a text color to prevent white text on white background dropdown items on Windows
-                        items={_.map(this.props.items, item => ({...item, color: themeColors.pickerOptionsTextColor}))}
-                        style={this.props.size === 'normal'
-                            ? styles.picker(this.props.isDisabled, this.props.backgroundColor)
-                            : styles.pickerSmall(this.props.backgroundColor)}
-                        useNativeAndroidPickerStyle={false}
-                        placeholder={this.placeholder}
-                        value={this.props.value}
-                        Icon={() => this.props.icon(this.props.size)}
-                        disabled={this.props.isDisabled}
-                        fixAndroidTouchableBug
-                        onOpen={this.enableHighlight}
-                        onClose={this.disableHighlight}
-                        textInputProps={{
-                            allowFontScaling: false,
-                        }}
-                        pickerProps={{
-                            ref: el => this.picker = el,
-                            onFocus: this.enableHighlight,
-                            onBlur: () => {
-                                this.disableHighlight();
-                                this.props.onBlur();
-                            },
-                            ...this.props.additionalPickerEvents(
-                                this.enableHighlight,
-                                (value, index) => {
-                                    this.onInputChange(value, index);
-                                    this.disableHighlight();
```

Comment:
removed disableHighlight function call to highlight color consistently across all option selections.